tests/jetstream.rs

```diff
@@ -516,3 +517,101 @@ fn jetstream_ordered() {
         assert_eq!(message.data, (i as i64).to_be_bytes());
     }
 }
+
+#[test]
+fn jetstream_pull_subscribe_batch() {
+    let s = util::run_server("tests/configs/jetstream.conf");
+    let nc = nats::Options::new()
+        .error_callback(|err| println!("error!: {}", err))
+        .connect(&s.client_url())
+        .unwrap();
+    let js = nats::jetstream::new(nc);
+
+    js.add_stream(&StreamConfig {
+        name: "TEST".to_string(),
+        subjects: vec!["foo".to_string()],
+        ..Default::default()
+    })
+    .unwrap();
+
+    js.stream_info("TEST").unwrap();
+    js.add_consumer(
+        "TEST",
+        ConsumerConfig {
+            durable_name: Some("CONSUMER".to_string()),
+            ..Default::default()
+        },
+    )
+    .unwrap();
+
+    for _ in 0..1000 {
+        js.publish("foo", b"lorem").unwrap();
+    }
+
+    let consumer = js.pull_subscribe("foo", "CONSUMER").unwrap();
+    let batch = consumer.batch(10).unwrap();
+
+    let mut i = 0;
+    for _ in batch {
+        i += 1;
+    }
+    assert_eq!(i, 10);
+
+    let batch = consumer.batch(10).unwrap();
+    for _ in batch {
+        i += 1;
+    }
+    assert_eq!(i, 20);
+}
+
+#[test]
+fn jetstream_pull_subscribe_timeout_batch() {
+    let s = util::run_server("tests/configs/jetstream.conf");
+    let nc = nats::Options::new()
+        .error_callback(|err| println!("error!: {}", err))
```

Comment:
In Go client if you do not provide one we default to printing to stderr, should do same here IMO.

Reply:
TL;DR: We can add it.
Tho I would like to have a clear note somewhere in the docs (maybe on connect?) that not specifying logs will make library log to stderr.

**Long story**:
Lot of production setups (especially those following Cloud Native approach) require all logs to be structured and JSON formatted (often following some standards like https://www.elastic.co/guide/en/ecs/master/ecs-log.html).
In NATS client we log data in plain text without user explicitly asking for it and we're not forcing them to handle those on their end.
That can cause log parsers to error. I did encounter it  in the past - Logstash was erroring when hitting those error logs and ignoring those messages in the result. Folks were blind looking at Kibana and not knowing what is happening in NATS client that it doesn't work as expected. 
It's nothing of high importance and user can easily handle that by specifying `error_callback` and handling logs there. They just need IMHO a clear info about that.
It just created unnecessary frustration which I would like to avoid as much as possible for NATS users across all libraries.

Reply:
Added a note.